packages/engine-api-client/package.json

```diff
@@ -61,6 +60,9 @@
     "chai-as-promised": "^7.1.1",
     "chai": "^4.3.6"
   },
+  "peerDependencies": {
+    "c-kzg": "^1.0.7"
```

Comment:
This rationale of this package is "abstract communication with an execution layer client". Thus it should not be aware even that c-kzg exists, else it's a bad abstraction.
- In this PR https://github.com/ChainSafe/lodestar/pull/4931 I refactor the role of the mock which IMO should not end up in the engine-api-client package as it has nothing to do with the goal stated above